src/mail/view/MailViewer.js

```diff
@@ -264,6 +264,11 @@ export class MailViewer {
 				m.redraw()
 			})
 
+		// Delay the display of the progress spinner in main body view for a short time to suppress it when just sanitizing
+		this._delayProgressSpinner = true
+		new Promise(resolve => setTimeout(resolve, 300))
```

Comment:
should use `Promise.delay` or just pass the callback directly to `setTimeout` pls

Reply:
Sure, don't really know how it got this convoluted...